packages/forms-web-app/src/controllers/full-appeal/submit-appeal/proposed-development-changed.js

```diff
@@ -24,7 +24,7 @@ const postProposedDevelopmentChanged = async (req, res) => {
 
   const descriptionDevelopmentCorrect = {
     isCorrect:
-      body['description-development-correct'] && body['description-development-correct'] === 'yes',
+      body['description-development-correct'] === 'yes',
     details:
```

Comment:
this is from another (merged) PR - rebase to prevent this happening

Reply:
resolved.